server/services/forms/schemas/record-interest.js

```diff
@@ -0,0 +1,84 @@
+const Form = require('../form-builder');
+const { Component, Choice } = require('../component-builder');
+
+function buildActionTypeChoiceArray(rawDataArray) {
+    if (rawDataArray.length > 0) {
+        return rawDataArray.map(el => Choice(el.typeInfo.actionLabel, el.typeInfo.uuid));
+    }
+    return [];
+}
+
+function buildVisibilityConditionsValuesString(interestTypeArray) {
+    return interestTypeArray
+        .map(appealType => appealType.typeInfo.uuid).join();
```

Comment:
Sorry @agraham19 can you change the naming on these please as well? To `interestType`